desktop/main.js

```diff
@@ -81,6 +81,8 @@ let downloadedVersion;
 // So we track preferredLocale separately via ELECTRON_EVENTS.LOCALE_UPDATED
 let preferredLocale = CONST.DEFAULT_LOCALE;
 
+const appProtocol = CONST.DEEPLINK_BASE_URL.replace('://', '');
```

Comment:
There's also a scheme in  [electronBuilder.config.js](https://github.com/Expensify/App/blob/3a9ce6fc59bb54d25749579be9b082c7600b235b/config/electronBuilder.config.js#L73)